package.json

```diff
@@ -103,6 +103,8 @@
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
     "release": "standard-version && vsce publish && git push --follow-tags",
+    "version": "standard-version && git push --follow-tags",
```

Comment:
- Where is `version` command used?
- I think `version` in npm scripts will be triggered when `npm version` run. See [npm pre--post-scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts). Is it intentional to do so, or not?
- Now `version` is almost the same to `release`, is it possible to update `release` command to `npm run version && vsce publish`?